RELEASING.md

```diff
@@ -0,0 +1,23 @@
+# Releasing
+
+1. Run the script `release.sh`:
+   ```bash
+   ./release.sh
```

Comment:
![image](https://github.com/RedMadRobot/gradle-version-catalogs/assets/18189767/0a24c326-9083-4e8e-8dd4-10b79b80261a)

Reply:
maybe put a default value for the current repository GITHUB_REPOSITORY in gradle.properties?